samples/extensions/TIBCO/function/condition/descriptor.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "condition",
+  "type": "flogo:function",
+  "version": "0.2.0",
+  "title": "Condition Functions",
+  "description": "Condition Functions",
+  "homepage": "https://github.com/TIBCOSoftware/tci-flogo/tree/master/samples/extensions/TIBCO/function/condition",
+  "functions": [
+    {
+      "name": "if",
+      "description": "return the second or the third parameter",
+      "example": "if true return second parameter, else return the third parameter",
+      "args": [
+        {
+          "name": "condition",
+          "type": "boolean"
+        },
+        {
+          "name": "any1",

```

Comment:
How about change the name to **trueArg** and **falseArg** which easy to understand first look.

Reply:
Much more intuitive